src/middlewared/middlewared/plugins/jail.py

```diff
@@ -407,22 +412,64 @@ def fstab(self, jail, options):
         """
         _, _, iocage = self.check_jail_existence(jail, skip=False)
 
-        action = options["action"].lower()
-        source = options["source"]
-        destination = options["destination"]
-        fstype = options["fstype"]
-        fsoptions = options["fsoptions"]
-        dump = options["dump"]
-        _pass = options["pass"]
-        index = options["index"]
+        verrors = ValidationErrors()
 
-        if action == "replace" and index is None:
+        action = options['action'].lower()
+        source = options.get('source')
+        if source:
+            if not os.path.exists(source):
+                verrors.add(
+                    'source',
+                    'Provided path for source does not exist'
+                )
+
+        destination = options.get('destination')
+        if destination:
+            destination = f'/{destination}' if destination[0] != '/' else destination
+            dst = f'{self.get_iocroot()}/jails/{jail}/root{destination}'
+            if 'mnt' in destination:
+                verrors.add(
+                    'destination',
+                    'Please specify destination path from root folder of jail'
+                    'e.g "/mnt/iocage/jails/btsync/root/" Path should be specified'
+                    'after this / now'
```

Comment:
We should be more clear and say `Destination should not include the jail path.`, also shouldn't you be doing spaces after for each line?

```
                    'Please specify destination path from root folder of jail'
                    ' e.g "/mnt/iocage/jails/btsync/root/" Path should be specified'
                    ' after this / now'
```